valhalla/meili/match_result.h

```diff
@@ -21,7 +21,7 @@ struct MatchResult {
   // Which edge this match point stays
   baldr::GraphId edgeid;
   // Percentage distance along the edge
-  double distance_along;
+  double percent_along;
```

Comment:
oh i see you just renamed this

Reply:
right.  I mean, its a percentage.  :).  Valhalla uses the naming convention 'percent_along' in a number of other places.  same context.